navigation2menu.js

```diff
@@ -26,7 +26,7 @@ for (const line of markdown.split('\n')) {
         continue;
     }
     const [_, name, pageRef] = matched;
-    const level = (indentPart.length - 2)/3;
+    const level = (indentPart.length - 2)/4;
```

Comment:
I've tested this on https://docs-qa.med-web.com/49-merge/eh/  and it looks good 👍